.travis.yml

```diff
@@ -28,6 +28,20 @@ install:
 
 jobs:
   include:
+  - stage: commons
+    install:
+    - sudo apt-get update
+    - sudo apt-get autoremove postgis*
+    - sudo apt-get autoremove postgresql*
+    - sudo apt-get install postgresql-9.3-postgis-2.1
+    before_script:
+    - psql -c "CREATE USER \"www-data\" WITH PASSWORD 'www-data';" -U postgres
+    - psql -c "CREATE DATABASE c2cgeoportal_tests WITH OWNER c2cgeoform;" -U postgres
+    - psql -d c2cgeoportal_tests -c "CREATE EXTENSION postgis;" -U postgres
+    script:
+    - cd commons
+    - make check test
```

Comment:
This is a good first point but is should use the same test infra as for geoportal, is it OK for you to merge like it and I will uniformise this?

Reply:
Yes, I'd like to merge this ASAP.

Reply:
Note we are not yet ready for docker and apache.